frontend/src/components/pages/ImportStoryPage.tsx

```diff
@@ -32,6 +50,85 @@ const ImportStoryPage = () => {
     },
   });
 
+  const onFileUpload = (e: any) => {
+    setStoryFile(e.target.files[0]);
+  };
+
+  const onFileDelete = () => {
+    setStoryFile(null);
+  };
+
+  const onDragEnter = (e: any) => {
+    e.preventDefault();
+    setDragOver(true);
+  };
+
+  const onDragLeave = (e: any) => {
+    e.preventDefault();
+    setDragOver(false);
+  };
+
+  const onDragOver = (e: any) => {
+    e.preventDefault();
+    if (
+      e.dataTransfer.items &&
+      e.dataTransfer.items.length === 1 &&
+      e.dataTransfer.items[0].kind === "file"
+    ) {
+      e.dataTransfer.dropEffect = "copy";
+    }
+  };
+
+  const onFileDrop = (e: any) => {
+    e.preventDefault();
+    setDragOver(false);
+    if (
+      e.dataTransfer.items &&
+      e.dataTransfer.items.length === 1 &&
+      e.dataTransfer.items[0].kind === "file"
+    ) {
+      const newStoryFile = e.dataTransfer.items[0].getAsFile();
+      if (
+        newStoryFile.type ===
+        "application/vnd.openxmlformats-officedocument.wordprocessingml.document"
+      ) {
+        setStoryFile(newStoryFile);
+      } else {
+        console.log("Incorrect file type");
+      }
+    } else if (e.dataTransfer.files && e.dataTransfer.files.length === 1) {
+      const newStoryFile = e.dataTransfer.files[0];
+      if (
+        newStoryFile.type ===
+        "application/vnd.openxmlformats-officedocument.wordprocessingml.document"
+      ) {
+        setStoryFile(newStoryFile);
+      }
+    }
+  };
+  const submitForm = async () => {
+    if (!(storyFile && title && description && level && youtubeLink)) {
+      window.alert("Please fill out all required fields.");
+      return;
+    }
+
+    const result = await importStory({
+      variables: {
+        storyFile,
+        storyDetails: {
+          title,
+          description,
+          youtubeLink,
+          level,
```

Comment:
Can you add the excluded languages as translated_languages in this call? Currently the column is null